File: app/src/main/java/com/task/data/remote/service/NewsService.java
package com.task.data.remote.service;

import com.task.data.remote.dto.NewsModel;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by AhmedEltaher on 5/12/2016
 */

public interface NewsService {
    @GET("topstories/v2/home.json")
    Call<NewsModel> fetchNews();
}

File: app/src/main/java/com/task/data/remote/service/NewsService.kt
package com.task.data.remote.service

import com.task.data.remote.dto.NewsModel
import retrofit2.Response
import retrofit2.http.GET

/**
 * Created by AhmedEltaher
 */

interface NewsService {
    @GET("topstories/v2/home.json?api-key=4rfwOLzLTWd1a5xixcPjwddAhw3p0eiF")
    suspend fun fetchNews(): Response<NewsModel>
}
